zbus/src/connection/raw/macos.rs

```diff
@@ -0,0 +1,21 @@
+#![cfg(target_os = "macos")]
+
+use crate::{addr::DBusAddr, process::run, Error, Result};
```

Comment:
> zb: add/rewrite raw stream connection 

Again, a huge bunch of changes with no details in the commit log of what's going on.

Reply:
ok